content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'setTimeout()'
+Description: 'setTimeout() allows users to execute callbacks after a period of time expressed in milliseconds'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+`setTimeout()` is a function serviced globally by the window object provided by the user's browser, it is not part of JavaScript because it is not a JavaScript function.
+
+## Syntax
+
+`setTimeout()` is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds **(1s = 1000ms)**, that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
+
+The provided sample code contains some examples of how setTimeout is composed:
+
+```pseudo
+setTimeout(callback)
+
+setTimeout(callback, delay)
+
+setTimeout(callback, delay, param)
+```
+
+## Usage example
+
+The following pieces of code will show some examples about the setTimeout implementation:
+
+```shell
+// Defining the function
+
+function sayHello() {
+  console.log('Hello from Codecademy');
+}
+
+// Calling the function using setTimeout.
+
+setTimeout(sayHello, 4000);
+```
+
+The output will be displayed after 4 seconds:
+
+```output
+Hello from Codecademy
+```
+
+An example using [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions) would be:
+
+```shell
+// Defining the function
+
+setTimeout(() => {
+  console.log('HelloWorld from Codecademy');
+}, 2000);
+```
+
+The output from the arrow function will be displayed after 2 seconds:
+
+```output
+HelloWorld from Codecademy
+```
+
+### Example using a parameters
```

Comment:
Change to AP style title

Reply:
also rephrase to example using parameters. Consistency is most important so if you want to break down the headers to "Example using <whatever specification>", I'd recommend changing the rest to fit that same structure. Change these to h2's instead of h3 for each header where you'll include an example.

Reply:
@yangc95 ready, please check

Reply:
Already commited some new titles, please check @yangc95